circuitbreaker/circuit.go

```diff
@@ -14,8 +15,6 @@ type Options func(breaker *CircuitBreaker)
 var (
   ErrCircuitOpen     = errors.New("http circuit breaker is open")
   ErrCircuitInternal = errors.New("internal error with circuit breaker")
-
-  noopHook = func() {}
```

Comment:
Seems not being used anywhere ?!

Reply:
Thanks a lot for contributing to this project. I wrote this implementation as a first draft, but it needs more refining and cleanup a bit. I really appreciate the time you took on this. I have not worked on this in a while for a lack of time. but I will in the near future. Please do not hesitate to contribute again if you have anything you want to show.

As for the linter, normally I use standard go tools (go fmt and go vet), but I forgot to setup my pre-commit hook last time I committed.